pkg/utils/util.go

```diff
@@ -53,6 +53,7 @@ type Options struct {
 	ToolchainApiUrl string
 	KeycloakUrl     string
 	OfflineToken    string
+	Workspace       string
```

Comment:
isn't the `workspace` value identical to the value of `userName`, i.e. the first parameter of the [`NewFramework`](https://github.com/redhat-appstudio/e2e-tests/pull/711/files#diff-b247c3614709818eed09ee97e1d6988bd26d0d11e7ad8269143bb84745e5b273R47) function? 

my idea is to use the `userName` value as a workspace name instead of introducing a new field to the `Options` struct